Embrace Unity iOS Interface/EmbraceManager.swift

```diff
@@ -0,0 +1,494 @@
+            try Embrace.client?.metadata.addProperty(key: key, value: value, lifespan: lifespan)
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func removeSessionProperty(key: String) -> Bool {
+        do {
+            try Embrace.client?.metadata.removeProperty(key: key, lifespan: .permanent)
+            try Embrace.client?.metadata.removeProperty(key: key, lifespan: .session)
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func logMessageWithSeverityAndProperties(
+        message: String,
+        severity: String,
+        properties: [String: String]) {
+            Embrace.client?.log(message,
+                severity: convertStringToLogSeverity(from: severity),
+                attributes: properties
+            )
+    }
+    
+    static func setUserAsPayer() -> Bool {
+        do {
+            try Embrace.client?.metadata.add(persona: "payer")
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func clearUserAsPayer() -> Bool {
+        do {
+            try Embrace.client?.metadata.remove(persona: "payer", lifespan: .session)
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func startView(viewName: String) -> String? {
+        let span = Embrace.client?.buildSpan(name: "emb-screen-view")
+            .setAttribute(key: "view.name", value: viewName)
+            .setAttribute(key: "emb.type", value: "ux.view")
+            .startSpan()
+        
+        guard let span else {
+            return nil
+        }
+        
+        return spanRepository.spanStarted(span: span)
+    }
+    
+    static func endView(spanId: String) -> Bool {
+        return stopSpan(spanId: spanId, errorCodeString: "", endTimeMs: 0.0)
+    }
+    
+    static func logNetworkRequest(url: String,
+                                  httpMethod: String,
+                                  startInMillis: Double,
+                                  endInMillis: Double,
+                                  bytesSent: Double,
+                                  bytesReceived: Double,
+                                  statusCode: Double,
+                                  error: String?) {
+        var attributes = [
+            "http.request.method": httpMethod.uppercased(),
+            "url.full": url
+        ]
+        
+        if statusCode >= 0 {
+            attributes["http.response.status_code"] = String(Int(statusCode))
+        }
+
+        if bytesSent >= 0 {
+            attributes["http.request.body.size"] = String(Int(bytesSent))
+        }
+
+        if bytesReceived >= 0 {
+            attributes["http.response.body.size"] = String(Int(bytesReceived))
+        }
+        
+        if let error {
+            attributes["error.message"] = error
+        }
```

Comment:
All the strings are exposed in the `EmbraceSemantics` targetin `SpanSemantics.NetworkRequest`:
```suggestion
        var attributes = [
            SpanSemantics.NetworkRequest.keyMethod: httpMethod.uppercased(),
            SpanSemantics.NetworkRequest.keyUrl: url
        ]
        
        if statusCode >= 0 {
            attributes[SpanSemantics.NetworkRequest.keyStatusCode] = String(Int(statusCode))
        }

        if bytesSent >= 0 {
            attributes[SpanSemantics.NetworkRequest.keyBodySize] = String(Int(bytesSent))
        }

        if bytesReceived >= 0 {
            attributes[SpanSemantics.NetworkRequest.keyResponseSize] = String(Int(bytesReceived))
        }
        
        if let error {
            attributes[SpanSemantics.NetworkRequest.keyErrorMessage] = error
        }
```